tox.ini

```diff
@@ -56,7 +56,8 @@ commands =
     rm -f docs/modules.rst
     make -C docs clean
     make -C docs html
-    python setup.py check --restructuredtext --strict
```

Comment:
`readme-renderer==35.0` dropped the legacy `check` command and suggests to switch using `twine` instead. https://github.com/pypa/readme_renderer/blob/main/CHANGES.rst#350-2022-04-19